16_wordle/main.go

```
@@ -46,18 +60,11 @@ func main() {
 
 func NewList(path string) (List, error) {
 	rawWordList, err := os.ReadFile(path)
```

You should return the error as soon as possible in most cases. In this case it would be here :)

---

Something like this?

---

Yes :)